dev/NavigationView/NavigationViewItemAutomationPeer.cpp

```diff
@@ -306,3 +313,49 @@ int32_t NavigationViewItemAutomationPeer::GetPositionOrSetCountInTopNavHelper(wi
 
     return returnValue;
 }
+
+bool NavigationViewItemAutomationPeer::IsSelected()
+{
+    if (auto nvi = Owner().try_as<winrt::NavigationViewItem>())
+    {
+        return nvi.IsSelected();
+    }
+    return false;
+}
+
+winrt::IRawElementProviderSimple NavigationViewItemAutomationPeer::SelectionContainer()
+{
+    winrt::IRawElementProviderSimple provider{ nullptr };
```

Comment:
>winrt::IRawElementProviderSimple provider{ nullptr }; [](start = 4, length = 53)

try to avoid scratch variable where possible here I would just return  where you set provider and return nullptr as the last line of the function.